akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/PersistentActorImpl.scala

```diff
@@ -5,25 +5,28 @@ package akka.persistence.typed.internal
 
 import akka.actor.ActorLogging
 import akka.actor.typed.internal.adapter.ActorContextAdapter
+import akka.actor.typed.scaladsl.ActorContext
 import akka.annotation.InternalApi
 import akka.persistence.journal.Tagged
-import akka.persistence.typed.scaladsl.{ PersistentBehavior, PersistentBehaviors }
+import akka.persistence.typed.scaladsl.PersistentBehaviors
 import akka.persistence.{ RecoveryCompleted, SaveSnapshotFailure, SaveSnapshotSuccess, SnapshotOffer, PersistentActor ⇒ UntypedPersistentActor }
 import akka.{ actor ⇒ a }
 
 /**
  * INTERNAL API
  */
+@Deprecated
 @InternalApi private[akka] object PersistentActorImpl {
 
   /**
-   * Stop the actor for passivation. `PoisonPill` does not work well
-   * with persistent actors.
+   * Stop the actor for passivation.
+   * `PoisonPill` does not work well  with persistent actors.
```

Comment:
not relevant for Typed, well you can technically send it to a typed actor but then you are doing nasty things